src/components/Calendar/Calendar.mdx

```diff
@@ -23,6 +24,9 @@ import { Calendar } from '@diconium/ui-toolbox';
 
 <Canvas of={CalendarStories.SpecificDateInThePast} />
 
+
+
```

Comment:
add one more story showing an example how to attach to `onPreviousClicked` and `onNextClicked`